AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginIntegrationTests/GraphQL/GraphQLWithUserPoolIntegrationTests/AuthDirective/GraphQLAuthDirectiveIntegrationTests.swift

```diff
@@ -49,7 +50,7 @@ class GraphQLAuthDirectiveIntegrationTests: XCTestCase {
 
     override func tearDown() {
         signOut()
-        await Amplify.reset()
+        // await Amplify.reset()
```

Comment:
please ignore this for now, upcoming changes to enable this test is to uncomment and add `async throws` to the teardown signature